src/components/SessionExpiry.jsx

```diff
@@ -0,0 +1,62 @@
+import React, {useState, useEffect} from "react";
+
+const SessionExpiry = () => {
+  const [timeLeft, setTimeLeft] = useState(600);
+  const [isReset, setIsReset] = useState(false)
+  const [isLoggedIn, setIsLoggedIn] = useState(true)
+
+  const resetTime = e => {
+    setIsReset(!isReset)
+  }
+
+  const displayTime = (timeLeft) => {
+    if(Math.floor(timeLeft % 60) < 10){
+      return Math.floor(timeLeft/60) + ":" + "0" + Math.floor(timeLeft % 60)
+    } else {
+      return Math.floor(timeLeft/60) + ":" +  Math.floor(timeLeft % 60)
+    }
+  }
+
+  useEffect(() => {
```

Comment:
essentially if we could move out the callable function, it could be unit tested.

Reply:
Not sure what you mean.  Remove it from the useEffect?

Reply:
the 1st function callback in `useEffect` could be named and moved outside. will help in testing later on better; so that whole thing could be say `const Foo = () => {...}` and then called from `useEffect(Foo)`

Reply:
I just pushed up that recommended change.